.circleci/config.yml

```diff
@@ -115,6 +116,29 @@ jobs:
       - run: make check
       - run: make checkscripts
 
+  build-binaries:
+    executor: go
+    environment:
+      <<: *COMMON_ENVS
+      GOPATH: /go
+      GO_TAGS: "ui codegen_generated"
+    steps:
+      - checkout
+      - run: apt-get update; apt-get install -y sudo unzip zip
+      - run: make deps
+      - install-protoc
+      - run: sudo -E PATH="$GOPATH/bin:/usr/local/go/bin:$PATH" make generate-structs
```

Comment:
We probably want to regenerate ui assets here as well, instead of the committed ui assets, potentially stale since last release.

Reply:
Unfortunately it looks like we don't have a reproducible UI build environment other than "use the `circleci/node:10-browsers` docker image", so I'm not sure what's involved in that. Given that we don't actually test the UI in e2e and I've added the `GO_TAGS=ui` mostly for convenience, how about I remove the `ui` tag so that it's 1:1 with the existing builds and circle back to that problem under a separate PR?

Reply:
Done in d2177b2